app/views/assignments/_form.html.erb

```diff
@@ -3,7 +3,7 @@
 
   <div class="form-inputs">
     <%= f.association :person,
-                collection: Person.active.where(:department => @requirement.event.departments) %>
+                collection: Person.active.where(:department => @requirement.event.departments).sort_by {|firstname, middleinitial, lastname| "#{firstname} #{middleinitial} #{lastname}"}.compact %>
```

Comment:
Not needed for this PR, but that's a lot of logic to have in the controller. We should move it in a later PR.

Reply:
I'm sorry, I thought this issue is what this PR is supposed to fix. Am I missing something?

Reply:
Don't worry about it.